Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -707,7 +707,7 @@ namespace mu2e {
     
     for (unsigned int i=0; i< _trigPaths.size(); ++i){
       string&path = _trigPaths.at(i);
-      if (trigNavig.accept(path)) _sumHist._hTrigInfo[15]->Fill((double)i);
+      if (trigNavig.accepted(path)) _sumHist._hTrigInfo[15]->Fill((double)i);
```

Comment:
Can we make the _sumHist a TH1I (integer) and avoid the cast here?

Reply:
In the near future, I wanted to improve this code for using the protonbunch intensity as weight. That's why I left TH1F. If we think it's a problem I can switch to TH1I